### DISCUSSION TITLE
Unable to deploy edge functions

### DISCUSSION THREAD

**Discussion by imbhargav5** [supabase/supabase#8967] [Questions] [answered]
I have been trying to deploy a simple edge function to my project but haven't been able to. It works fine on local. The steps I followed were

1. `supabase login` and used my access token to login to the cli
2. `supabase functions new hello-world`
3. `supabase functions deploy hello-world --project-ref myprojectref`

However, I run into this error message. 

```bash
Bundling hello-world
Error: Failed to create a new Function on the Supabase project: {"message":"Could not obtain service for project 7278","errorEventId":"8b2d20264f024b0485c773098e4bf480"}
```

Has anyone else ever faced this? 

Thanks very much!

**Comment by imbhargav5** [comment, marked as answer]
Solved by the supabase support staff.

**Comment by Klustre** [comment]
Had the same error. After trying to deploy the function multiple times it succeeded.
